src/components/Audiences/processDestinationsFactory.js

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import createDestinations from "../../utils/createDestinations";
+import { isObject, isNonEmptyString, cookie } from "../../utils";
+
+export default ({ destinations, config, logger }) => {
+  const urlDestinations = destinations.reduce((arr, dest) => {
+    if (isObject(dest) && dest.type === "url" && isObject(dest.spec)) {
+      arr.push({
+        url: dest.spec.url,
+        hideReferrer: dest.spec.hideReferrer
+      });
+    }
```

Comment:
Is there any reason to not just `arr.push(dest.spec)`? If not, and considering my comment above, I think we could shrink this all down to:

```js
const urlDestinations = destinations.filter(dest => dest.type === "url").map(dest => dest.spec);
```